examples/field_examples/viking_graben_line12/proc/README.md

```diff
@@ -0,0 +1,18 @@
+# Viking Graben Line 12: processing
+
+The processing is done using [Madagascar](https://www.reproducibility.org/wiki/Main_Page) open source software. Thus we assume that Madagascar is present on the system.
```

Comment:
This is quite nice. Too bas it can't be done in julia would have been a really nice package. So most of this is for RTM  and/or not used right?

Reply:
Hum so there is a julia API

https://github.com/ahay/src/tree/master/api/julia

not sure if it actually works though

Reply:
Not sure if this is actually needed to rewrite it using Julia API. This will require some time to rework and Madagascar is very common to be used with Python. I understand that JUDI mostly relies on Julia but Python is required as well.